src/main/java/org/folio/inventory/instanceingress/handler/InstanceIngressUpdateEventHandler.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.inventory.instanceingress.handler;
+
+import java.util.concurrent.CompletableFuture;
+import org.folio.rest.jaxrs.model.InstanceIngressEvent;
+
+public class InstanceIngressUpdateEventHandler implements InstanceIngressEventHandler {
+
+  @Override
+  public CompletableFuture<InstanceIngressEvent> handle(InstanceIngressEvent instanceIngressEvent) {
+    // to be implemented in MODINV-1008
+    return null;
```

Comment:
suggest to return empty action future instead of null

Reply:
what's the matter? the next task in our team will be MODINV-1008.
But not a big deal, changed